pyproject.toml

```diff
@@ -41,4 +42,4 @@ sunspecdemo = 'sunspecdemo.cli:cli'
 
 [build-system]
 requires = ["setuptools", "poetry-core>=1.0.0", "poetry-dynamic-versioning",]
-build-backend = "poetry.core.masonry.api"
```

Comment:
did the masonry.api do anything that we need?

Reply:
The `poetry_dynamic_versioning.backend` is a wrapper (they call it a "patch") for `poetry.core.masonry.api` to allow poetry dynamic versioning to function.